src/index.ts

```diff
@@ -1,9 +1,10 @@
+import { EventEmitter } from 'events';
```

Comment:
Pretty sure this is unused and might have been left over from the merge?

Reply:
Probably, will do it when I get home.

Reply:
done